src/grid-bots/multi-vendor-multi-contract.py

```diff
@@ -7,146 +7,106 @@
 import json
 import numpy as np
 import pandas as pd
-import scipy
-import talib
 from scipy.interpolate import interp1d
-from talib import RSI
+from talib import RSI, MACD
 import asyncio
 import time
 
+TIME_LOOKUP = {
+  '1m':  60_000,
+  '5m' : 60_000 * 5,
+  '15m': 60_000 * 15,
```

Comment:
15m won't be supported soonish so could remove.

Reply:
Removed